shared/js/background/utils.es6.js

```diff
@@ -209,6 +209,15 @@ function brokenListIndex (url, list) {
     })
 }
 
+function isUrlExcludedByFeature (url, feature) {
```

Comment:
Can this match the naming of the other functions please?

`isFeatureBrokenForURL` perhaps or flip the logic for `isFeatureEnabledForURL`